### PR TITLE
Dockerfiles: refactoring, update README

### PR DESCRIPTION
**Pull request by stokowski**
Changes for runner, prerunner, transform-hub:
- add docker-entrypoints
- optimize Dockerfiles
- update README

### CODE REVIEW

File: packages/pre-runner/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.12.2",
   "description": "This package is part of Scramjet Transform Hub. The package identifies the sequences and returns the information to back STH.",
   "scripts": {
-    "build:docker": "docker build -t ${SCRAMJET_INT_REPO_PATH}scramjetorg/pre-runner:$npm_package_version .",
+    "build:docker": "docker build -t scramjetorg/pre-runner:$npm_package_version .",
```
Comment (daro1337) [inline packages/pre-runner/package.json:6]:
```suggestion
    "build:docker": "docker build -t scramjetorg/pre-runner:$(jq -r .version package.json) .",
```

File: packages/runner/package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
   "bin": "bin/start-runner.js",
   "scripts": {
     "build": "tsc -p tsconfig.build.json",
-    "build:docker": "docker build -t ${SCRAMJET_INT_REPO_PATH}scramjetorg/runner:$npm_package_version -f Dockerfile ../../",
+    "build:docker": "docker build -t scramjetorg/runner:$npm_package_version -f Dockerfile ../../",
```
Comment (daro1337) [inline packages/runner/package.json:9]:
```suggestion
    "build:docker": "docker build -t scramjetorg/runner:$(jq -r .version package.json) -f Dockerfile ../../",
```

File: packages/sth/package.json [hunk 3]
```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "start": "ts-node ./src/index",
     "build": "tsc -p tsconfig.build.json",
-    "build:docker": "docker build -t ${SCRAMJET_INT_REPO_PATH}scramjetorg/sth:$npm_package_version -f Dockerfile ../../",
+    "build:docker": "docker build -t scramjetorg/sth:$npm_package_version -f Dockerfile ../../",
```
Comment (daro1337) [inline packages/sth/package.json:13]:
```suggestion
    "build:docker": "docker build -t scramjetorg/sth:$(jq -r .version package.json) -f Dockerfile ../../",
```

File: README.md [hunk 4]
```diff
@@ -298,10 +298,11 @@ To run Hub without docker-compose:
 ```bash
 docker run -d --init \
   --name scramjet-hub \
+  --init \
```
[the remark this answers is not in the document]
Comment (stokowski) [reply README.md:302]: fixed

File: packages/pre-runner/package.json (hunk 1 above)
Comment (daro1337) [reply packages/pre-runner/package.json:6]: My mistake, I don't know why I had in my head that it was about the node version.

**Review by daro1337 — APPROVED** [review]

**Pull request merged**